src/Pyz/Yves/ShopUi/Theme/default/components/molecules/custom-select/custom-select.scss

```diff
@@ -227,6 +233,10 @@
             vertical-align: middle;
             cursor: pointer;
 
+            &--focus {
+                @include helper-outline;
+            }
```

Comment:
Do we need it?

Reply:
Yes. With this we highlight entire dropdown